exams/urls.py

```diff
@@ -1,12 +1,16 @@
 """URLs for exams app"""
 from django.conf.urls import url
 
-from exams.views import PearsonCallbackRedirectView
+from exams.views import (
+    PearsonCallbackRedirectView,
+    PearsonSSO,
+)
 
 urlpatterns = [
     url(
         r'^pearson/(?P<status>success|error|timeout|logout)/?$',
         PearsonCallbackRedirectView.as_view(),
         name='pearson_callback'
     ),
+    url(r'^api/v0/pearson/$', PearsonSSO.as_view(), name='pearson_sso_api'),
```

Comment:
Could you namespace this a little more (e.g. `api/v0/pearson/sso/`) in case we end up adding other pearson apis down the road?